docs/source/api/accounts.md

```diff
@@ -227,9 +227,11 @@ First, add the service configuration package:
 meteor add service-configuration
 ```
 
-Then, in your app (this example is for the Weebo service):
+Then, in your app (this example is for the Weebo service), import `ServiceConfiguration` inside your `server/main.js` file:
```

Comment:
I don't think we need to specify which file on server it should be, just that it needs to happen on server.

Reply:
sure, it makes sense. I've just updated it :)